src/main/java/com/iexec/core/task/TaskService.java

```diff
@@ -71,6 +73,10 @@ public TaskService(TaskRepository taskRepository,
         return Optional.empty();
     }
 
+    public boolean doesTaskNeedSGX(Task task) {
```

Comment:
We will need that method in the worker too, can you please use instead (& update) this mehod https://github.com/iExecBlockchainComputing/iexec-common/blob/002ed4651e87983e375f3f5ceff13657342b8177/src/main/java/com/iexec/common/tee/TeeUtils.java#L14 pleaase ?

Reply:
done